install-deps.sh

```diff
@@ -414,6 +414,8 @@ else
         echo "Using zypper to install dependencies"
         zypp_install="zypper --gpg-auto-import-keys --non-interactive install --no-recommends"
         $SUDO $zypp_install systemd-rpm-macros
+        # python header files are needed to install inside the virtualenv (eg. xmlsec needs the header files)
+        $SUDO $zypp_install python-devel python3-devel
```

Comment:
i think we already install `python%{python3_pkgversion}-devel`, so presumably, python3-devel should be installed by
```sh
$SUDO $zypp_install $(rpmspec -q --buildrequires $DIR/ceph.spec)
```
and since we dropped python2 support in master, we don't need to install `python-devel` anymore.